python/cli/prelude_cli/views/iam.py

```diff
@@ -52,12 +52,15 @@ def update_account(controller, mode, company):
 
 
 @iam.command('account')
+@click.option('--noqueue', help='hide your detect queue', is_flag=True)
 @click.pass_obj
 @handle_api_error
-def describe_account(controller):
+def describe_account(controller, noqueue):
```

Comment:
Remove this change (unrelated)

Reply:
I don't love the change, otherwise I'd have mentally skipped it :) 

I'm working on my flaws!